src/main/webapp/WEB-INF/tags/master.tag

```diff
@@ -13,7 +13,7 @@
       <img src="${pageContext.servletContext.contextPath}/images/logo.svg"/>
       PhoneShop
     </a>
-    <jsp:include page="/minicart"/>
+    <%@ include file="/WEB-INF/pages/minicart.jsp"%>
```

Comment:
or it is also possible to do with <jsp:include page
the reason why it didn;t work is that you filled wrong path to jsp in previous version